### PR TITLE
fix: add type fix to subscription link to fix 'prepare' script

### PR DESCRIPTION
**Pull request by david-mcafee**
*Issue #, if available:*

*Description of changes:*
Currently, the `prepare` script fails because of a type error. This PR resolves this issue.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/aws-appsync-subscription-link/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-appsync-subscription-link",
-  "version": "3.0.8",
+  "version": "3.0.9",
```
Comment (elorzafe) [inline packages/aws-appsync-subscription-link/package.json:3]: lerna should do it automatically

**Review by elorzafe — CHANGES_REQUESTED** [review]
I would revert the version bump, lerna will do it when releasing this

File: packages/aws-appsync-subscription-link/package.json (hunk 1 above)
Comment (david-mcafee) [reply packages/aws-appsync-subscription-link/package.json:3]: Will remove!

**Review by elorzafe — APPROVED** [review]
LGTM! thanks @david-mcafee 🌮

**Pull request merged**